### ISSUE TITLE
cdktf diff is not showing plan output when ran on Terraform Cloud

### ISSUE THREAD

**Issue by daniel-laszlo** [hashicorp/terraform-cdk#2234]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

Python 3.9
CDKTF 0.13.2
Terraform Cloud

### Affected Resource(s)

`cdktf diff`

### Debug Output

```
$ cdktf diff STACK                 
[2022-10-31T15:31:40.275] [DEBUG] default - Initializing error reporting
[2022-10-31T15:31:40.282] [DEBUG] default - Collecting environment information for error reporting
[2022-10-31T15:31:40.299] [DEBUG] default - {
  "terraform_version": "1.3.2",
  "platform": "darwin_arm64",
  "provider_selections": {},
  "terraform_outdated": true
}

[2022-10-31T15:31:40.299] [DEBUG] default - {
  "terraform_version": "1.3.2",
  "platform": "darwin_arm64",
  "provider_selections": {},
  "terraform_outdated": true
}

[2022-10-31T15:31:40.300] [DEBUG] default - v16.16.0

[2022-10-31T15:31:40.313] [DEBUG] default - v16.16.0

[2022-10-31T15:31:40.351] [DEBUG] default - Terraform v1.3.2
on darwin_arm64

[2022-10-31T15:31:40.352] [DEBUG] default - 
Your version of Terraform is out of date! The latest version
is 1.3.3. You can update by downloading from https://www.terraform.io/downloads.html


⠇  Synthesizing
[2022-10-31T15:31:41.890] [DEBUG] default - Name: constructs

[2022-10-31T15:31:41.890] [DEBUG] default - Version: 10.1.145
Summary: A programming model for software-defined state
Home-page: https://github.com/aws/constructs
Author: Amazon Web Services
Author-email: 
License: Apache-2.0

[2022-10-31T15:31:41.891] [DEBUG] default - Location: /Users/daniel.laszlo/.local/share/virtualenvs/cdktf--05tE73m/lib/python3.9/site-packages
Requires: jsii, publication, typeguard
Required-by: cdktf, cdktf-cdktf-provider-gitlab

[2022-10-31T15:31:41.908] [DEBUG] default - Name: cdktf

[2022-10-31T15:31:41.908] [DEBUG] default - Version: 0.13.2
Summary: Cloud Development Kit for Terraform
Home-page: https://github.com/hashicorp/terraform-cdk
Author: HashiCorp

[2022-10-31T15:31:41.909] [DEBUG] default - Name: jsii
Version: 1.70.0
Summary: Python client for jsii runtime
Home-page: https://github.com/aws/jsii
Author: Amazon Web Services
Author-email: 
License: Apache-2.0
Location: /Users/daniel.laszlo/.local/share/virtualenvs/cdktf--05tE73m/lib/python3.9/site-packages
Requires: attrs, cattrs, publication, python-dateutil, typeguard, typing-extensions

[2022-10-31T15:31:41.909] [DEBUG] default - Author-email: 
License: MPL-2.0
Location: /Users/daniel.laszlo/.local/share/virtualenvs/cdktf--05tE73m/lib/python3.9/site-packages
Requires: constructs, jsii, publication, typeguard
Required-by: cdktf-cdktf-provider-gitlab


⠏  Synthesizing
[2022-10-31T15:31:41.991] [DEBUG] default - Failed to run pip --version: Error: spawn pip ENOENT

⠸  Synthesizing


STACK  Creating Terraform Cloud configuration version

STACK  Created Terraform Cloud configuration version cv-XXXXXXXXXXXXXXXX

⠧  Processing
[2022-10-31T15:31:49.203] [DEBUG] default - Could not remove .terraform folder Error: ENOENT: no such file or directory, stat 'PATH/cdktf/cdktf.out/stacks/STACK/.terraform'
    at Object.statSync (node:fs:1551:3)
    at __node_internal_ (node:internal/fs/utils:799:8)
    at Object.rmSync (node:fs:1226:13)
    at Ff.removeLocalTerraformDirectory (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:458:9727)
    at Ff.init (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:458:3256)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Ff.n.value (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:458:666)
    at async FM.initalizeTerraform (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:458:14385)
    at async /usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:458:14791
    at async FM.run (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:458:14465) {
  errno: -2,
  syscall: 'stat',
  code: 'ENOENT',
  path: 'PATH/cdktf/cdktf.out/stacks/STACK/.terraform'
STACK  Zipping up the directory cdktf.out/stacks/STACK

STACK  Uploading the directory to Terraform Cloud

STACK  Uploaded the directory to Terraform Cloud

STACK  Waiting for configuration version to become ready...



STACK  Creating speculative Terraform Cloud run

STACK  Created speculative Terraform Cloud run: https://app.terraform.io/app/ORGANIZATION/workspaces/WORKSPACE/runs/run-XXXXXXXXXXXXXXXX

STACK  Speculative Terraform Cloud run done

STACK  Getting plan output

⠋  Processing

$  # Command returned at this point
```

### Expected Behavior

Show list of additions, removals, modifications - this works for `cdktf deploy` for some reason.

### Actual Behavior

No output at all.

**Comment by DanielMSchmidt** [comment]
This is not really a bug, but the expected behaviour right now. We might change the behaviour though when we switch out our current Terraform Cloud implementation by using the terraform cli.

**Comment by DanielMSchmidt** [comment]
Terraform CLI does show the plan, so #1955 should fix this

**Comment by DanielMSchmidt** [comment]
Fixed in #1955
